internal/security/auth.go

```diff
@@ -25,6 +25,7 @@ func SetAuthConfig(canaryConfig *config.CanaryConfig, saramaConfig *sarama.Confi
 			return errors.New("SASL password must be specified")
 		}
 		saramaConfig.Net.SASL.Enable = true
+		saramaConfig.Net.SASL.Version = sarama.SASLHandshakeV1
```

Comment:
should that be configurable?

Reply:
version V0 doesn't provide information about a SASL authentication failure while it's provided by V1 which is supported by ages. I don't see any value supporting V0 imho.

Reply:
See https://cwiki.apache.org/confluence/display/KAFKA/Kafka+APIs

Reply:
Supported since Kafka 1.0 in fact.